### PR TITLE
Clarify zeropoint computation

### PR DESCRIPTION
**Pull request by aboucaud**
This no longer needs a reference mag value. That reference is already comprised in the `ab_zeropoint` value from speclite.

### CODE REVIEW

**Comment by mpaillassa** [comment]
Thanks for going into this, it's better !
Though when I run the script I get:
```
Rubin filters are not available in speclite
```
while we have renamed `Rubin` to `LSST` everywhere ? Do you have the same behavior ? @aboucaud

**Comment by aboucaud** [comment]
> ```
> Rubin filters are not available in speclite
> ```
> 
> while we have renamed `Rubin` to `LSST` everywhere ? Do you have the same behavior ? @aboucaud

Nope, it runs smoothly for me
```sh
$ python scripts/check_zeropoints.py

Checking the zeropoints with speclite
-------------------------------------

-- Euclid_VIS --	(Euclid in speclite)

filters |  speclite |  galcheat
------- | --------- | ---------
  VIS   | 25.92 mag | 25.91 mag


CFHTLS filters are not available in speclite


COSMOS filters are not available in speclite


-- LSST --	(lsst2016 in speclite)

filters |  speclite |  galcheat
------- | --------- | ---------
   u    | 26.44 mag | 26.40 mag
   g    | 28.29 mag | 28.26 mag
   r    | 28.13 mag | 28.10 mag
   i    | 27.81 mag | 27.78 mag
   z    | 27.42 mag | 27.39 mag
   y    | 26.59 mag | 26.56 mag


-- DES --	(decam2014 in speclite)

filters |  speclite |  galcheat
------- | --------- | ---------
   g    | 26.72 mag | 26.72 mag
   r    | 26.99 mag | 26.99 mag
   i    | 26.86 mag | 26.86 mag
   z    | 26.58 mag | 26.58 mag


-- HSC --	(hsc2017 in speclite)

filters |  speclite |  galcheat
------- | --------- | ---------
   g    | 28.90 mag | 28.90 mag
   r    | 28.86 mag | 28.86 mag
   i    | 28.61 mag | 28.61 mag
   z    | 27.68 mag | 27.68 mag
   y    | 27.34 mag | 27.33 mag
```

**Comment by ismael-mendoza** [comment]
Make sure to add something to the docs related to using ab_magnitudes e.g. mag2counts

**Comment by mpaillassa** [comment]
I fixed my installation ! 
I was writing the small change in the doc to mention that zeropoints are AB zeropoints and should thus be used with AB magnitudes. And I was wondering if it would not be even better to rename `zeropoint` to `ab_zeropoint` to make things really clear. What do you think ? @aboucaud @ismael-mendoza

**Comment by ismael-mendoza** [comment]
> And I was wondering if it would not be even better to rename zeropoint to ab_zeropoint to make things really clear. What do you think ?

Thanks @mpaillassa - Yes I agree that would make it really clear!

**Comment by aboucaud** [comment]
I am just afraid that `ab_zeropoint` is starting with `ab` and prevents users from finding interactively the zeropoint.
I would suggest using `zeropoint_ab` OR leaving it like that with a proper documentation.

**Comment by ismael-mendoza** [comment]
I'm ok with `zeropoint_ab`, I agree that might be better for finding it

**Comment by aboucaud** [comment]
@ismael-mendoza @mpaillassa let me know if the zeropoint description I wrote makes sense to you. This is the closest I could get to the actual computation on speclite.

File: docs/parameters.md [hunk 1]
```diff
@@ -40,10 +40,11 @@ The following parameters are required in the YAML file describing any filter of
...
 | psf_fwhm             |   float   |     arcsec      | Average full width at half-maximum (FWHM) of the point spread function (PSF) over the filter.                                                                                                                                                                                                 |
-| zeropoint            |   float   |       mag       | The zeropoint is the magnitude of an object that produces one electron per second on the detector. It is computed for a given filter, using the [`speclite`][speclite] library with a classical atmosphere, at the airmass given by the survey `zeropoint_airmass`.                           |
+| zeropoint            |   float   |       mag       | The zeropoint value is a reference magnitude computed as the magnitude in the AB system (see [this reference][abmag_ref]) collected in one second and on the survey effective area. It takes into account the filter bandwidth and a classical atmosphere for ground based surveys, using the [`speclite`][speclite] library, at the airmass given by the survey `zeropoint_airmass`.                           |
```
Comment (mpaillassa) [inline docs/parameters.md:43]: I would say the "instrument effective area" not to be confused with the total area covered by the survey, even though I doubt that someone would misunderstand this.

Then I don't know if it is necessary to mention that speclite is not used for all surveys.

Comment (ismael-mendoza) [reply docs/parameters.md:43]: I agree with @mpaillassa on these two points. Otherwise looks clear to me

Comment (ismael-mendoza) [reply docs/parameters.md:43]: what do we do with the other surveys?

**Review by ismael-mendoza — APPROVED** [review]
LGTM, just a small comment echoing @mpaillassa comment.

File: docs/parameters.md (hunk 1 above)
Comment (mpaillassa) [reply docs/parameters.md:43]: For the other surveys we find other references (this includes CFHTLS and COSMOS).

Comment (aboucaud) [reply docs/parameters.md:43]: Yep, I will default to the references section then.

File: docs/parameters.md [hunk 2]
```diff
@@ -40,10 +40,11 @@ The following parameters are required in the YAML file describing any filter of
...
-| zeropoint            |   float   |       mag       | The zeropoint is the magnitude of an object that produces one electron per second on the detector. It is computed for a given filter, using the [`speclite`][speclite] library with a classical atmosphere, at the airmass given by the survey `zeropoint_airmass`.                           |
+| zeropoint            |   float   |       mag       | The zeropoint value for most surveys is a reference magnitude computed as the magnitude in the AB system (see [this reference][abmag_ref]) collected on the instrument effective area in one second. It takes into account the filter bandwidth and a classical atmosphere for ground based surveys, using the [`speclite`][speclite] library, at the airmass given by the survey `zeropoint_airmass`. For the remaining surveys is extracted from the references (see `references` parameter).                          |
```
Comment (ismael-mendoza) [inline docs/parameters.md:43]:
```suggestion
| zeropoint            |   float   |       mag       | The zeropoint value for most surveys is a reference magnitude computed as the magnitude in the AB system (see [this reference][abmag_ref]) collected on the instrument effective area in one second. It takes into account the filter bandwidth and a classical atmosphere for ground based surveys, using the [`speclite`][speclite] library, at the airmass given by the survey `zeropoint_airmass`. For the remaining surveys it is extracted from corresponding the references (see `references` parameter).                          |
```

**Pull request merged**